src/assets/styles/_utilities.scss

```diff
@@ -31,3 +31,13 @@
   border: none;
   cursor: pointer;
 }
+html:not(.is-fullscreen) .app-content-container {
+  padding-top: var(--AppHeaderHeight, initial);
+  display: grid;
+  grid-template-columns: var(--AppSidebarWidth) 1fr;
+}
+
+html:not(.is-fullscreen) .app-main-content {
+  padding: var(--AppGap);
+}
```

Comment:
**Suggestion**: I think these styles should be defined as close as possible to where they’re being used and that’s App.vue and AppView.vue.

Reply:
Hmm yeah not totally sure, they aren't being used in either of those components anymore, plus if we decided we wanted to put them into `RouteView` that is putting our specific application concerns into "framework components", and if we need to do this more and more, then `RouteView` is only going to get full of unrelated CSS.

I did wonder where was the best place to put them and saw that we had utility CSS things in this file, this thing being a 'utility'. Do you have any other ideas where this could go?

Reply:
> Hmm yeah not totally sure, they aren't being used in either of those components anymore

But ... they are? App.vue uses `app-content-container` on line 14 and AppView.vue uses `app-main-content` on line 4.

And those respectively are the key elements of the selectors in play here. Those components are exactly where those rules should live. The `html:not(.is-fullscreen)` is irrelevant here as this CSS is for the elements targeted by the key selector. `html:not(.is-fullscreen)` is just modifying when they apply.

Also, these are really not utility rules. This file is for re-usable utilities.

Reply:
Oh I see what you mean now! Sorry I thought you were talking about `is-fullscreen`, one sec lemme have another think.